pkg/apis/performanceprofile/v1/performanceprofile_types.go

```diff
@@ -78,7 +81,7 @@ type CPUSet string
 // CPU defines a set of CPU related features.
 type CPU struct {
 	// Reserved defines a set of CPUs that will not be used for any container workloads initiated by kubelet.
-	Reserved *CPUSet `json:"reserved,omitempty"`
+	Reserved *CPUSet `json:"reserved"`
```

Comment:
this is probably correct but should not be done in this PR. Or if it is, is not clear why and should be done on a separate commit with proper rationale

Reply:
done